src/prog_models/prognostics_model.py

```diff
@@ -1147,7 +1147,9 @@ def generate_surrogate(self, load_functions, first_output = None, threshold_keys
             'inputs': self.inputs,
             'outputs': self.outputs,
             'events': self.events,
-            'stability_tol': 1e-05
+            'stability_tol': 1e-05,
+            'data_add_noise': True,
```

Comment:
Why "data_add_noise" and not simply "add_noise"?
is there a reason for "data" to be there?

Reply:
Great point, I ended up putting these together into "training_noise"